### PR TITLE
Add a recipe for PyQt4

### PR DESCRIPTION
**Pull request by ccordoba12**
This one should pass in all operating systems because it's not that time consuming ;-)

Pinging @ocefpaf about it.

### CODE REVIEW

File: recipes/pyqt/bld.bat [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+:: This solution copied from http://around-the-corner.typepad.com/adn/2015/05/building-sip-and-pyqt-for-maya-2016.html
+
+pushd pylupdate
+del moc_translator.cpp 2> nul
+del moc_translator.obj 2> nul
+moc.exe -o moc_translator.cpp translator.h
+if errorlevel 1 exit 1
+popd
+
+jom
```
Comment (ocefpaf) [inline recipes/pyqt/bld.bat]: Will this work in `conda-forge`? Not sure what is `jom` install.

File: recipes/pyqt/build.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+$PYTHON configure.py \
+          --verbose \
+          --confirm-license \
+          --bindir=$PREFIX/bin \
+          --destdir=$SP_DIR \
+          --qmake=$PREFIX/bin/qmake
+
+make
+make install
```
Comment (ocefpaf) [inline recipes/pyqt/build.sh]: Is there a `make check`?

File: recipes/pyqt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - qt 4.8.*
+    - sip 4.18
+    - jom  # [win]
```
Comment (ocefpaf) [inline recipes/pyqt/meta.yaml]: OK. It is available on `defaults` so it will work.

**Comment by ocefpaf** [comment]
Thanks for putting this together @ccordoba12!

File: recipes/pyqt/build.sh (hunk 2 above)
Comment (ccordoba12) [inline recipes/pyqt/build.sh]: We can verify that on the feedstock. I have very little time for this now ;-)

In any case, a much better test is the one in `run_test.py`, which verifies that the a gui application is started correctly.

**Comment by ccordoba12** [comment]
No problem :-)

@ocefpaf, could you cancel the builds? I forgot to add `recipe-maintainers` ;-)

File: recipes/pyqt/build.sh (hunk 2 above)
Comment (ocefpaf) [inline recipes/pyqt/build.sh]: Sure. No problem.

**Comment by ocefpaf** [comment]
> @ocefpaf, could you cancel the builds?

Done. Feel free to add me there as a maintainer.

**Comment by ccordoba12** [comment]
Done.

**Comment by ccordoba12** [comment]
Thanks! I hope this solves all Qt4 related problems for conda-forge.

@ceholden, could you try to build QGIS with this new package and see how it goes? Thanks!

**Comment by ceholden** [comment]
@ccordoba12 Thank you very much! Compiling `qscintilla2` and `qgis` is working as expected again.

**Pull request merged**